handlers/generic_file_upload_ce_attachment_v1/README.md

```diff
@@ -0,0 +1,46 @@
+# Kinetic Request CE Attachment Upload With API
```

Comment:
In the node.xml and CHANGELOG.md files you named this handler `Generic File Upload Ce Attachment`